code/modules/surgery/organs/stomach/external/_external_organs.dm

```diff
@@ -170,8 +170,10 @@
 	dna_block = DNA_FRILLS_BLOCK
 
 /obj/item/organ/external/frills/can_draw_on_bodypart(mob/living/carbon/human/human)
-	if(!(human.head?.flags_inv & HIDEEARS))
+	if(human.head?.flags_inv & ~HIDEEARS)
```

Comment:
![image](https://user-images.githubusercontent.com/35135081/124943411-50ab0700-dfc1-11eb-903a-459d433635c4.png)

These two statements are not equivalent. Are you sure this is what you meant to do?

![image](https://user-images.githubusercontent.com/35135081/124943488-61f41380-dfc1-11eb-874e-081ca985e457.png)

Reply:
'~' isn't just a fancy inverse? Whatever I reverted it to how it was but with the 'return FALSE', since I at least know how that works

Reply:
You *could've* done `x ^ ~y`, but it's not as readable.